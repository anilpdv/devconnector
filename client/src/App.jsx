import React from "react";
import Navbar from "./components/layout/navbar";
import Footer from "./components/layout/footer";
import Landing from "./components/layout/landing";
import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
import "./App.css";
import Register from "./components/auth/Register";
import Login from "./components/auth/Login";

function App() {
  return (
    <div className="App">
      <Router>
        <Navbar />
        <Switch>
          <Route exact path="/" render={props => <Landing />} />
          <Route path="/register" render={props => <Register />} />
          <Route path="/login" render={props => <Login />} />
        </Switch>

        <Footer />
      </Router>
    </div>
  );
}

export default App;
