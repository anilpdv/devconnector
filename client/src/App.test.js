import React from "react";
import { shallow, mount } from "enzyme";
import App from "./App";

describe("App Test", () => {
  it("should render without crashing", () => {
    const app = shallow(<App />);
    expect(app).toMatchSnapshot();
  });

  it("should render every children without crashing", () => {
    const app = mount(<App />);
    expect(app).toMatchSnapshot();
  });
});
