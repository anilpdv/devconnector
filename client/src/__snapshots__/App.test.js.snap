// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Test should render every children without crashing 1`] = `ReactWrapper {}`;

exports[`App Test should render without crashing 1`] = `ShallowWrapper {}`;
